packages/build/package.json

```diff
@@ -45,7 +44,6 @@
     "lb-mocha": "./bin/run-mocha.js",
     "lb-nyc": "./bin/run-nyc.js",
     "lb-dist": "./bin/select-dist.js",
-    "lb-apidocs": "./bin/generate-apidocs.js",
```

Comment:
IMO, removing `lb-apidocs` is a breaking change requiring a semver-major release. Please add `BREAKING CHANGE` footer to the commit message and explain LB users how to deal with this change - see https://github.com/strongloop/loopback-next/blob/0d9698ab3bbad2754736b15b8e8880187c167c32/docs/site/DEVELOPING.md#describe-incompatibilites-for-release-notes.

I have few breaking changes that I would like to introduce in `@loopback/build` too. Let's coordinate the efforts so that we can release them all together in a single major version.